vpc-sample/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Use the official Python image.
```

Comment:
Sample is missing a README: while helloworld-shell skipped one, this sample is very complicated  and a few notes about what's going on seems necessary. TODO links to tutorials.